### PR TITLE
Updating template for Exection Container compatibility

### PR DESCRIPTION
**Pull request by twerthi**
---

# Background

I've developed an Execution Container for Liquibase (not yet published).  This PR makes a minor update for container compatibility and does not affect current implementation for backwards compatibility.

# Results

When using the execution container, it will dynamically collect additional database drivers where necessary

## Before

If the user selected a database technology in which the driver did not ship with the Liquibase product, the execution would fail.

## After

The template will download the appropriate database driver based on what was selected.

# Pre-requisites

- [X] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [X] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [X] Parameter names should not start with `$`
- [X] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [X] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [X] The best practices documented [here](https://github.com/OctopusDeploy/Library/wiki/Best-Practices) have been applied
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

### CODE REVIEW

**Comment by benPearce1** [comment]
@twerthi I noticed that we have the following code in the script:

```
if ([string]::IsNullOrEmpty($liquibaseExecutablePath)) {
 # Assign root
 $liquibaseExecutablePath = $PSSCriptRoot
}
```

Could we do something similar to this when running in a container, to avoid the user having to provide an arbitrary value:
```
if ([string]::IsNullOrEmpty($liquibaseExecutablePath) -and $env:IsContainer) {
 # Assign root
 $liquibaseExecutablePath = '/'
}
```

**Comment by twerthi** [comment]
Not a bad idea, thanks for the suggestion :)

**Comment by twerthi** [comment]
Updated.

File: step-templates/liquibase-run-command.json [hunk 1]
```diff
@@ -193,7 +193,7 @@
       "Id": "cae0020d-595b-407c-a96b-02f7a6bc9788",
       "Name": "liquibaseExecutablePath",
       "Label": "Executable file path",
-      "HelpText": "File path to the Liquibase executable.",
+      "HelpText": "File path to the Liquibase executable.\n\nIf using the `octopuslabs/liquibase-workertools` Execution Container, use the value of `/`",
```
Comment (benPearce1) [inline step-templates/liquibase-run-command.json:196]: If this is no longer required, we could revert this.

**Review by benPearce1 — APPROVED** [review]
👍  left one further comment

**Pull request merged**